docs/source/cheat_sheet_py3.rst

```diff
@@ -235,113 +235,116 @@ Classes
 .. code-block:: python
 
    class MyClass:
-       # The __init__ method doesn't return anything, so it gets return
-       # type None just like any other method that doesn't return anything.
+       # You can optionally declare instance variables in the class body
+       attr: int
+       # This is an instance variable with a default value
+       charge_percent: int = 100
+
+       # The "__init__" method doesn't return anything, so it gets return
+       # type "None" just like any other method that doesn't return anything
        def __init__(self) -> None:
            ...
-       # For instance methods, omit `self`.
+
+       # For instance methods, omit type for "self"
        def my_method(self, num: int, str1: str) -> str:
            return num * str1
 
+   # User-defined classes are valid as types in annotations
+   x: MyClass = MyClass()
 
+   # You can use the ClassVar annotation to declare a class variable
+   class Car:
+       seats: ClassVar[int] = 4
+       passengers: ClassVar[List[str]]
+
+   # You can also declare the type of an attribute in "__init__"
+   class Box:
+       def __init__(self) -> None:
+           self.items: List[str] = []
 
-   # User-defined classes are written with just their own names.
-   x = MyClass() # type: MyClass
 
+Coroutines and asyncio
+**********************
 
-Other stuff
-***********
+See :ref:`async-and-await` for the full detail on typing coroutines and asynchronous code.
+
+.. code-block:: python
+
+   import asyncio
+   from typing import Generator, Any
+
+   # A Python 3.5+ coroutine is typed like a normal function
+   async def countdown35(tag: str, count: int) -> str:
+       while count > 0:
+           print('T-minus {} ({})'.format(count, tag))
+           await asyncio.sleep(0.1)
+           count -= 1
+       return "Blastoff!"
+
+   async def async2(obj: object) -> str:
+       return "placeholder"
+
+   # A generator-based coroutine created with @asyncio.coroutine should have a
```

Comment:
Python 3.4 EoL is in 9 months, also I doubt there are many Python 3.4 users out there, I would completely remove this part, and instead maybe add an example of async generator (3.6+).